versions.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2018, Google Inc.
+# Copyright (c) 2019, Google Inc.
```

Comment:
Google LLC

Reply:
@aav66 
I just have changed the year. Should I use this one?

```
# Copyright 2019 Google LLC.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```